port/freebsdamd64/omrsignal_context.h

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2016 IBM Corp. and others
```

Comment:
The copyright header should have the current year: `Copyright (c) 2016, 2018 IBM Corp. and others`.